scripts/crosstool-NG.sh.in

```diff
@@ -534,7 +534,7 @@ if [ -z "${CT_RESTART}" ]; then
     fi
     # Use the number of processors+1 when automatically setting the number of
     # parallel jobs.  Fall back to 1 if the host doesn't use GLIBC.
-    AUTO_JOBS=$((`getconf _NPROCESSORS_ONLN 2> /dev/null || echo 0` + 1))
+    AUTO_JOBS=$((`getconf _NPROCESSORS_ONLN 2> /dev/null || sysctl -n hw.ncpu 2>/dev/null || echo 0` + 1))
```

Comment:
Do you mean which command to use to determine the number of processors should be determined by configure.ac, or that the number of processors should be determined by configure.ac?